docs/index.html

```diff
@@ -15,7 +15,44 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
-
+	
+	<!-- AGLS Metadata -->
+	<link rel="schema.DCTERMS" href="http://purl.org/dc/terms/" /> 
+	<link rel="schema.AGLSTERMS" href="http://www.agls.gov.au/agls/terms/" />
+	<meta name="DCTERMS.title" content="Home page template - Australian Government Design System" />
+	<meta name="DCTERMS.identifier" content="https://designsystem.gov.au/templates/home/" />
+	<meta name="DCTERMS.creator" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=designsystem@dta.gov.au" />
+	<meta name="DCTERMS.publisher" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=designsystem@dta.gov.au" />
+	<meta name="DCTERMS.description" content="Home pages explain your product or service and help users find what they are looking for" />
+	<meta name="DCTERMS.modified" content="scheme=DCTERMS.ISO8601; 2019-08-06T11:36:29+10:00" />
+	<meta name="DCTERMS.subject" content="design system, templates" />		
+	<meta name="DCTERMS.language" content="DCTERMS.RFC4646; en-AU" />
+	<meta name="DCTERMS.type" content="Home page template" />
+	<meta name="DCTERMS.coverage" content="Australia" />
+	<meta name="DCTERMS.spatial" content="Australia" />
+	<meta name="AGLSTERMS.documentType" content="text" />
+	<meta name="msapplication-TileColor" content="#ffffff" />
+	<meta name="theme-color" content="#ffffff" />
+	<meta name="robots" content="index, follow" />
+	<meta name="author" content="Digital Transformation Agency" /> 
+	<meta name="description" content="Home page template - Australian Government Design System" />
+	
+	<!-- TWITTER CARD -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Home page template - Australian Government Design System" />
+	<meta name="twitter:site" content="@DTA" />
+	<meta name="twitter:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg" />
```

Comment:
Should we host these assets in the starter repo? Right now if we make a change to the DS repo, it could break things in this repo

Reply:
Good point - I'd suggest aiming to make the starter repo as self-contained as possible. Adding assets to the repo and making URL references relative could be a good way to start.

Reply:
Yep good point

We should do 
https://govau.github.io/design-system-starter/assets/favicons/designsystem.jpg